contrib/completion/zsh/_docker

```diff
@@ -666,6 +666,7 @@ __docker_subcommand() {
                 "($help)*--log-opt=[Log driver specific options]:log driver options:__docker_log_options" \
                 "($help)--mtu=[Set the containers network MTU]:mtu:(0 576 1420 1500 9000)" \
                 "($help -p --pidfile)"{-p=,--pidfile=}"[Path to use for daemon PID file]:PID file:_files" \
+                "($help)--raw-logs[Full timestamps w/o ANSI coloring]" \
```

Comment:
any reason not to use the full word `without`? Is it getting too long for an 80 char term?